src/test/java/com/gojek/beast/sink/BqIntegrationTest.java

```diff
@@ -82,6 +85,7 @@ public void setUp() throws Exception {
         gcsSinkHandler = new OOBErrorHandler(errorWriter);
         final BlobId blobId = BlobId.of("test-integ-godata", "test-integ-godata-dlq/beast/testfile");
         final BlobInfo objectInfo = BlobInfo.newBuilder(blobId).setContentType("text/plain").build();
+        appConfig = ConfigFactory.create(AppConfig.class, System.getProperties());
```

Comment:
lets not create appConfig if its not being used

Reply:
Again, It is getting used on line number 40 if the message is null.